e0works/MyDiary-front
# #162361663 users should be able to delete a dairy entry

[Delivers #162361663]

#### What does this PR do?
- implement delete entry functionality
#### Description of Task to be completed?
- create action
- create a reducer
- add deleteEntry method to Dashboard component

#### How should this be manually tested?
- login to the app
- click or on delete button on a diary entry
- you should be prompted to confirm a delete

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#162361663](https://www.pivotaltracker.com/story/show/162361663, "delete diary entry")

#### Screenshots (if appropriate)
NA

#### Questions:
